### ISSUE TITLE
Nuxt 3 support

### ISSUE THREAD

**Issue by HelloAlexPan** [WilliamDASILVA/nuxt-stripe-module#45]
Hey @WilliamDASILVA, would love to know whether there's plans to support Nuxt 3 and how the community can help.

**Comment by kiketordera** [comment]
Yeah, specially now that Vue3, Nuxt3, and this month Vuetify3 are available as stable versions

**Comment by khalidzamer** [comment]
@WilliamDASILVA  Nuxt3 is still in RC, will it support this Stripe module?

**Comment by StefanVDWeide** [comment]
@WilliamDASILVA I would love to help you out making this module compatible with Nuxt 3. Now that [Medusa JS](https://medusajs.com/) is gaining more popularity as an open-source Shopify alternative it would be great to be able to build a storefront with Nuxt 3. 

Do you maybe have a roadmap of changes that need to be made to this module?

**Comment by jpsala** [comment]
Guys, it's not working in nuxt3 for me. Do you have some tips?

**Comment by flozero** [comment]
Hey team Its a really old thread but a friend of mine @fuentesloic and I are actually working on a nuxt 3 version for stripe :). Will update this issue as soon as its online.

We will try to keep the same api as much as possible

**Comment by flozero** [comment]
Quick update it is on npm and we have created the PR to have it in nuxt modules :) 

https://github.com/fuentesloic/nuxt3-stripe

https://github.com/nuxt/modules/pull/677

Have fun coding please don't hesitate to ask for features we are actively working on it for now

**Comment by StefanVDWeide** [comment]
Awesome! Will keep an eye on your repo and hope it will be included in the Nuxt Modules list. I think we can consider this one abandoned unfortunately.
